guides/micronaut-k8s/api/src/main/resources/application.yml

```diff
@@ -1,3 +1,6 @@
+micronaut:
+  application:
+    name: api
```

Comment:
this should not be necessary @timyates since it is defined in bootstrap.yml

Reply:
Yes, but the applications when you create them

It's not necessary but it's a lot more obvious in the guide as to what you have to do